### PR TITLE
making none listed a lighter gray in available bidders list comments

### PR DESCRIPTION
**Pull request by SidelineCory24**

### CODE REVIEW

File: src/Components/AvailableBidder/AvailableBidderRow/AvailableBidderRow.jsx [hunk 1]
```diff
@@ -118,7 +118,7 @@ const AvailableBidderRow = (props) => {
     ted: formattedTed,
     current_post: getCustomLocation(),
     cdo: cdo ? getCDO() : NO_CDO,
-    comments: get(bidder, 'available_bidder_details.comments') || NO_COMMENTS,
+    comments: get(bidder, 'available_bidder_details.comments') || <text className="no-comments">{NO_COMMENTS}</text>,
```
Comment (SidelineCory24) [inline src/Components/AvailableBidder/AvailableBidderRow/AvailableBidderRow.jsx:121]: Is it cleaner if this is a variable instead?

Comment (mjoyce91) [reply src/Components/AvailableBidder/AvailableBidderRow/AvailableBidderRow.jsx:121]: I think this approach is fine, but the problem with wrapping this in an html element is that this happens when you open the modal:

Screen Shot 2021-04-20 at 12 32 23 PM

**Comment by mjoyce91** [comment]
We should also be able to make use of `aria-disabled="true"` to get around the color contrast check

File: src/Components/AvailableBidder/AvailableBidderRow/AvailableBidderRow.jsx [hunk 2]
```diff
@@ -172,9 +172,14 @@ const AvailableBidderRow = (props) => {
   return (
     <tr className={getTRClass()}>
       {
-        keys(sections).map(i => (
-          <td key={i}>{sections[i]}</td>
-        ))
+        keys(sections).map(i => {
+          if (i === 'comments' && sections[i] === NO_COMMENTS) {
```
Comment (elizabeth-jimenez) [inline src/Components/AvailableBidder/AvailableBidderRow/AvailableBidderRow.jsx:176]: Nice catch on changing "None listed" to NO_COMMENTS var 🦜

**Review by elizabeth-jimenez — APPROVED** [review]
🍰 🧁 👍 Looks good!

**Pull request merged**